staging/src/k8s.io/api/core/v1/generated.pb.go

```diff
@@ -7317,7 +7332,6 @@ func (m *PodIP) MarshalTo(dAtA []byte) (int, error) {
 	return i, nil
 }
 
->>>>>>> 5868685... generated items for the modified types (podIPInfo => PodIP)
```

Comment:
This merge conflict bit should be removed in the commit it was introduced in, not here...